### ISSUE TITLE
Shortcut to customize lists

### ISSUE THREAD

**Issue by ajtruckle** [ajtruckle/meeting-schedule-assistant-support#25]
**Is your feature request related to a problem? Please describe.**
It has always been possible to [customize](https://www.publictalksoftware.co.uk/help-msa/mwb-editor-customize-lists.html) various drop-down lists in the _Meeting Editor_. 

![image](https://github.com/ajtruckle/meeting-schedule-assistant-support/assets/7985331/b56ebb4a-8ab5-47e5-9052-f1c400495b95)

But this feature is not accessible when a popup window is active (eg. the [Assignment Material](https://www.publictalksoftware.co.uk/help-msa/mwb-editor-section3-material.html) window):

![image](https://github.com/ajtruckle/meeting-schedule-assistant-support/assets/7985331/8bd17336-698f-45f4-a431-f7979d372740)

**Describe the solution you'd like**
It would be handy if we could **add** to the _Student_ / _Method_ lists from _within_ the **Materials** window. The required changes needed for the GUI could be agreed to make it easier.

**Additional context**
Other places in the _Meeting Editor_ might possibly benefit from this feature

**Comment by cengizu** [comment]
The drop-down lists in the **Materials** window are already editable. This means that the user can choose default values or enter a custom content. These custom values entered by the user are saved somewhere and stay permanently. 

![image](https://github.com/ajtruckle/meeting-schedule-assistant-support/assets/6536357/dec70d2b-01cd-4d06-bcd6-8751bdf81bdb)

It would be practical if these values could be associated with the corresponding tabs of the Customize Lists. This would allow users to get things done without having to open Customize Lists separately in advance. Customize Lists will continue to exist.

**Comment by ajtruckle** [comment]
@cengizu 
> It would be practical if these values could be associated with the corresponding tabs of the Customize Lists. This would allow users to get things done without having to open Customize Lists separately in advance. Customize Lists will continue to exist.

I agree with your logic with respects to the **Student** drop-list. But I have reservations concerning the other drop-lists. For example, I would personally want to have the ability to add specific Material phrases to the drop-list. I would not necessarily want every distinct phrase that I use automatically added to the list. Just specific ones.

**Comment by ajtruckle** [comment]
@cengizu 

Good morning! I wondered if we could come up with a specific 16 x 16 icon as a shortcut to the customize lists window. We would show it on the right of these relevant combos in the editor:

### Note

![Image](https://github.com/user-attachments/assets/5a346ccb-3cdd-45dd-8522-e83025382ae9)

### Theme

![Image](https://github.com/user-attachments/assets/f4a35c92-42ac-4d0d-be4e-57fc6cd66bcb)

### Method

![Image](https://github.com/user-attachments/assets/3541d76a-c85f-408b-a482-8ffeeb086f00)

### Filter

![Image](https://github.com/user-attachments/assets/7c576dc6-11d7-496f-94f4-d3925e9a0ab8)

### Student

![Image](https://github.com/user-attachments/assets/e673972d-8ed5-4cbf-bab0-c7f4b6b10527)

### Material

![Image](https://github.com/user-attachments/assets/218d1447-e65e-47f7-a9a0-09bc95415244)

I suppose we could use the pencil icon.

**Comment by cengizu** [comment]
Here is a pencil icon (16x16) that you can try:
https://www.shareicon.net/modify-82355

**Comment by ajtruckle** [comment]
Hmm! I can't seem to download that icon:

![Image](https://github.com/user-attachments/assets/77881d1c-339d-4fe0-b25b-c64000fa3213)

Maybe we can just use our existing pencil icon. Eg:

![Image](https://github.com/user-attachments/assets/045a6fd0-69b5-4c54-8447-aaff15d82dff)

**Comment by ajtruckle** [comment]
> What if you right click the image and click "save image as..."

That works! And I think it looks cleaner in the editor:

![Image](https://github.com/user-attachments/assets/55e66649-e0cd-4edf-991d-2ff11aee61d9)

**Comment by ajtruckle** [comment]
My progress so far …

### Added Pencil buttons to the **Assignment History** and **Meeting Editor** windows:

![Image](https://github.com/user-attachments/assets/24ff4d5e-96c6-4ab5-9c54-8163f0061080)

![Image](https://github.com/user-attachments/assets/252d9942-077a-4077-a7ed-0b3bb11b2c0e)

![Image](https://github.com/user-attachments/assets/58a5f9c9-2e4c-4ac0-b0cf-605791ed3a4e)

I still need to work on the **Student Material** window.

I just had a thought - how about including **Delete from list** to the menu? It would  enabled if the current text is in the drop-list and is one of your own added phrases.

**Comment by ajtruckle** [comment]
I have updated the control to disable all other radio choices when editing in this mode:

![Image](https://github.com/user-attachments/assets/8f755261-e576-41be-bf6b-bd0bb228fec5)

Also updated the Student Material window:

![Image](https://github.com/user-attachments/assets/8ba7f283-6f94-4b7c-928c-907a0641557a)
